api/__init__.py

```diff
@@ -0,0 +1,18 @@
+from flask_restx import Api
+
+from .todo import api as todo_namespace
```

Comment:
Is TODO going to be a function of ValDB or is this just a piece of flask restx example? It it's the latter, please remove it.